main.go

```diff
@@ -65,7 +68,11 @@ func parseTemplates(templatePaths []string, leftLimiter string, rightLimiter str
 			return fmt.Errorf("failed to create target dir for %q: %w", templatePath, err)
 		}
 		targetFile, _ := os.Create(targetPath)
-		err = t.Execute(targetFile, secrets)
+		err = t.Execute(targetFile, struct {
+			Secrets map[string]map[string]string
+		}{
+			Secrets: secrets,
```

Comment:
The examples in `examples/k8s` and the README still use just `{{ .secret1 }}`, don't they need to use `{{ .Secrets.secret1.subkey }}` as keys now?